README.md

```diff
@@ -390,7 +392,7 @@ const argv = {};
 const config = require('./webpack.config.js');
 
 serve(argv, { config }).then((server) => {
-  server.on('listening', ({ server, options }) => {
```

Comment:
Why this change here ?

Reply:
I explained this both in the first comment and in the commit message:

> `listening` event will no longer be emmited after the serve() promise is resolved: https://github.com/webpack-contrib/webpack-serve/releases/tag/v2.0.0

I changed the example code to `'build-started'` event because it's the next best starting place for the reader to understand how to use the event hooks.